colournaming/templates/index.html

```diff
@@ -278,7 +278,7 @@ <h2 id="contact">
         <a class="github social-link" href="https://github.com/colournaming/colournaming"><!-- GitHub Logo --></a>
         <a class="email social-link" href="mailto:dmilonas@gmail.com"></a>
     </div>
-    <p class="lined-text">Made in London & Berlin</p>
+    <p class="lined-text">Made in London &amp Berlin</p>
```

Comment:
Still missing a semicolon it should be `&amp;` ([see here for a guide](https://www.w3schools.com/html/html_entities.asp)).